velox/core/QueryConfig.h

```diff
@@ -175,6 +175,13 @@ class QueryConfig {
   /// output rows.
   static constexpr const char* kMaxOutputBatchRows = "max_output_batch_rows";
 
+  /// Min number of rows to be returned by operators from
+  /// MergingVectorOutput::getOutput. It is used when an estimate of average
+  /// row size is not known. When the estimate of average row size is known,
+  /// kMinOutputBatchBytes is used.
```

Comment:
kMinOutputBatchBytes doesn't seem to exist.

Reply:
Remove it.